data_structures/read_group.wdl

```diff
@@ -180,7 +199,8 @@ task validate_read_group {
             if [[ ~{read_group.ID} =~ ^~{id_pattern}$ ]] \
             || [[ ~{read_group.ID} =~ ~{restrictive_pattern} ]]
             then
-                >&2 echo "ID (~{read_group.ID}) must not match pattern ~{id_pattern} nor ~{restrictive_pattern}"
+                >&2 echo "ID (~{read_group.ID}) must not match patterns:"
```

Comment:
I'm fairly certain that `pattern` should be singular because of the use of `nor`.

Reply:
We could also remove `nor` and replace it with some other delimiter.

Reply:
Updated